build/commands/lib/test.js

```diff
@@ -145,6 +154,9 @@ const runTests = (passthroughArgs, suite, buildConfig, options) => {
   }
 
   if (options.output) {
+    const outputPath = path.join(config.srcDir, options.output)
+    // Clear previous output
+    deleteFile(outputPath)
     braveArgs.push('--gtest_output=xml:' + options.output)
```

Comment:
wait. shouldn't this point to a `{testSuite}.xml` file?

Reply:
This is set before we loop over the suites. I am not actually sure why we even set this here since we override it later in the loop.